Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "merk"
 description = "Merkle key/value store"
-version = "2.0.0"
+version = "2.0.0-oo"
```

Comment:
Where does `oo` comes from?

Reply:
@hansl In order to carry two versions of merk in the many-rs PR, the versions must be different.  Arguably, this means we should be bumping the version _number_ on each PR merge, but we haven't previously performed this convention.